lib/ansible/plugins/connection/network_cli.py

```diff
@@ -349,13 +699,6 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
             )
         self.queue_message('log', 'network_os is set to %s' % self._network_os)
 
-    @property
-    def paramiko_conn(self):
-        if self._paramiko_conn is None:
-            self._paramiko_conn = connection_loader.get('paramiko', self._play_context, '/dev/null')
-            self._paramiko_conn.set_options(direct={'look_for_keys': not bool(self._play_context.password and not self._play_context.private_key_file)})
```

Comment:
This logic is not presently preserved, and the option has been added to network_cli.